bcgov/cas-registration
# Feat/307 activity pulp paper

Addresses bcgov/cas-reporting#307. This adds the _Pulp and Paper production_ activity.

## Changes 🚧

- Adds backend config/migration for _Pulp and Paper production_ activity.
- Adds frontend page and components for _Pulp and Paper production_ activity.

## To test 🔬

- `make reset_db` to apply latest migrations.
- spin up the front, backend and postgres.
- Login, then navigate to `http://localhost:3000/reporting/reports/1/facilities/1/activities/pulp_and_paper_production`, click through the available form fields. 
  - The `Gas type` dropdown options should change options available in the methodology field.
  - The `Methodology` dropdown options spawn additional fields, depending on the option selected.
